SYCL/InvokeSimd/call_vadd_1d.cpp

```diff
@@ -0,0 +1,195 @@
+// TODO: enable on Windows once driver is ready
+// REQUIRES: gpu && linux
+// UNSUPPORTED: cuda || hip
+//
+// Check that full compilation works:
+// RUN: %clangxx -fsycl -fno-sycl-device-code-split-esimd -Xclang -fsycl-allow-func-ptr %s -o %t.out
+// RUN: env IGC_VCSaveStackCallLinkage=1 IGC_VCDirectCallsOnly=1 %GPU_RUN_PLACEHOLDER %t.out
+// Now device-compile:
+// RUN: %clangxx -fsycl -Xclang -fsycl-allow-func-ptr -fsycl-device-only -S %s -o %t.ll
+// ... then run post-link on the compilation result, checking output IR:
+// ... some modules had to be split, '-ir-output-only' can't be used
+// RUN: sycl-post-link -lower-esimd -S %t.ll -o %t.xxx.ll
```

Comment:
Such tests which invoke sycl-post-link should normally go to the test/tools/sycl-post-link directory in the llvm repo, and use .ll as input